packages/create-pantheon-decoupled-kit/src/types.ts

```diff
@@ -56,9 +56,15 @@ export interface DecoupledKitGenerator<
 	/**
 	 * Identifies a generators compatible CMS(s).
 	 */
-	cmsType: 'wp' | 'drupal' | 'any';
+	cmsType: DrupalCMS | WordpressCMS | 'any';
 }
 
+/**
+ * Valid CMS Type Options
+ */
+export type DrupalCMS = 'd9' | 'd10' | 'drupal';
+export type WordpressCMS = 'wp' | 'wordpress';
```

Comment:
It might make sense to use a namespace here like so

```ts
/**
 * Valid CMS Type Options
 */
export type CMSType = {
  Drupal: 'd9' | 'd10' | 'drupal';
  WordPress: 'wp' | 'wordpress';
}
```